code/modules/mob/dead/observer/observer.dm

```diff
@@ -146,7 +146,8 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	show_data_huds()
 
 	SSpoints_of_interest.make_point_of_interest(src)
-	ADD_TRAIT(src, TRAIT_HEAR_THROUGH_DARKNESS, ref(src))
+	ADD_TRAIT(src, TRAIT_HEAR_THROUGH_DARKNESS, INNATE_TRAIT)
+	ADD_TRAIT(src, TRAIT_GOOD_HEARING, INNATE_TRAIT)
```

Comment:
Sssooo I added this as a replacement for the `isobserver` check in `Hear` (because `istype(src)` check evil, traits good) but I just realized. It was an `isobserver` check in the middle of `living/Hear` 

I'll leave this trait here to indicate the intent but I'm pretty sure it's dead code?